pennylane/transforms/qcut.py

```diff
@@ -364,6 +365,174 @@ def graph_to_tape(graph: MultiDiGraph) -> QuantumTape:
     return tape
 
 
+def _get_measurements(
+    group: Sequence[Operator], measurements: Sequence[MeasurementProcess]
+) -> List[MeasurementProcess]:
+    """Pairs each observable in ``group`` with the fixed circuit ``measurements``.
+
+    Only a single fixed measurement of an expectation value is currently supported.
+
+    Args:
+        group (Sequence[Operator]): a collection of qubit-wise commuting observables
```

Comment:
```suggestion
        group (Sequence[Operator]): a collection of observables
```
Not strictly required for this function